src/index.html

```diff
@@ -6,8 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+        src="./images/imac.jpeg"
```

Comment:
not fixed
<img width="551" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/98764641/cbc7cb45-cd1a-400c-9bee-0a24e9b93011">